File: fix.py
import os
filepath = 'web-admin/src/services/realtimeStore.ts'
with open(filepath, 'r', encoding='utf-8') as f:
    c = f.read()
c = c.replace('\\\'', "'")
with open(filepath, 'w', encoding='utf-8') as f:
    f.write(c)

File: fix.js
const fs = require('fs');
const file = 'web-admin/src/pages/FieldLogsPage.tsx';
let code = fs.readFileSync(file, 'utf8');

code = code.replace(
  "          }}\n        />",
  "          }}\n          onDelete={async (id) => {\n            await deleteFieldLog(id);\n            setEditLog(null);\n          }}\n        />"
);

fs.writeFileSync(file, code);
console.log('Added onDelete prop');
